src/request/middleware/url-encoded-body.ts

```diff
@@ -9,7 +9,7 @@ export default function parseUrlEncodedBody(
 ): InterfaceRequest {
   const { headers, body } = request
 
-  return headers['content-type'] === 'application/x-www-form-urlencoded' &&
+  return headers['content-type'].split(';').shift() === 'application/x-www-form-urlencoded' &&
```

Comment:
Let's add a truth-y check before we try to split, just in case the content-type header is undefined. E.g. `headers['content-type'] && headers['content-type'].split()...`